doc/guides/dts/index.rst

```diff
@@ -383,5 +395,39 @@ information extraction phase and are matched to device tree nodes via the
 compatible property.
 
 
+Include files generation
+************************
+
+At build, after board .dts file has been processed by dtc (device tree compiler),
+a frdm_k64f.dts_compiled file is generated under the zephyr directory.
+This file is parsed by python dts parsing script, and thanks to YAML bindings it
+generates, in the include/generated directory, an include file named
+generated_dts_board_unfixed.h.
+This include file holds all the information extracted from the dts file under
+the format specified by the YAML bindings:
+
+.. code-block:: c
+
+   /* gpio_keys */
+   #define DT_GPIO_KEYS_0		1
+
+   /* button_0 */
+   #define DT_GPIO_KEYS_BUTTON_0_GPIO_CONTROLLER	"GPIO_2"
+   #define DT_GPIO_KEYS_BUTTON_0_GPIO_FLAGS	0
+   #define DT_GPIO_KEYS_BUTTON_0_GPIO_PIN		6
+   #define DT_GPIO_KEYS_BUTTON_0_LABEL		"User SW2"
+   #define DT_GPIO_KEYS_SW1_GPIO_CONTROLLER	DT_GPIO_KEYS_BUTTON_0_GPIO_CONTROLLER
+   #define DT_GPIO_KEYS_SW1_GPIO_FLAGS		DT_GPIO_KEYS_BUTTON_0_GPIO_FLAGS
+   #define DT_GPIO_KEYS_SW1_GPIO_PIN		DT_GPIO_KEYS_BUTTON_0_GPIO_PIN
+   #define DT_GPIO_KEYS_SW1_LABEL			DT_GPIO_KEYS_BUTTON_0_LABEL
+   #define SW1_GPIO_CONTROLLER			DT_GPIO_KEYS_BUTTON_0_GPIO_CONTROLLER
+   #define SW1_GPIO_FLAGS				DT_GPIO_KEYS_BUTTON_0_GPIO_FLAGS
+   #define SW1_GPIO_PIN				DT_GPIO_KEYS_BUTTON_0_GPIO_PIN
+   #define SW1_LABEL				DT_GPIO_KEYS_BUTTON_0_LABEL
+
+Additionally, a concatenated file named generated_dts_board_fixups.h is
+generated in the same directory using all board related fixup files.
+
+These both files are then combined into a generated_dts_board.conf file.
```

Comment:
> These two files are included by both files are then combined into a generated_dts_board.conf file.

No it's not.

.conf corresponds to generated_dts_board_unfixed.h.

You could however say that:

"The source file generated_dts_board.h should be included by C source files to access DT information as it in turn includes generated_dts_board_unfixed.h and generated_dts_board_fixups.h.

Reply:
My bad. Txs for proposal.